packages/card/MediaCard.jsx

```diff
@@ -66,16 +71,19 @@ export default function MediaCard({
           </div>
         )}
         {title && (
-          <h2 className="su-text-[35px] su-mb-5 su-leading[110%] md:su-text-[40px] lg:su-text-[43px]">
+          <h2 className="su-text-[3.5rem] su-mb-5 su-leading-tight md:su-text-[4rem] lg:su-text-[4.3rem]">
             {liveUrl && (
+              // eslint-disable-next-line jsx-a11y/control-has-associated-label
               <a
                 href={liveUrl}
-                className="su-text-black su-transition dark:su-text-white hocus:su-text-digital-red dark:hocus:su-text-dark-mode-red"
+                className="su-group su-text-black su-transition dark:su-text-white hocus:su-text-digital-red dark:hocus:su-text-dark-mode-red hocus:su-underline"
               >
                 <XssSafeContent content={title} elementType="span" />
-                <span className="su-inline-block *:su-inline-block *:su-w-42 *:su-h-42 su-ml-[-42px] su-translate-x-[42px]">
-                  <ExternalArrow />
-                </span>
+                {isRealExternalLink && (
+                  <span className="su-inline-block *:su-inline-block *:su-size-1em su--ml-1em su-translate-x-1em group-hocus:su-translate-x-[1.1em] group-hocus:su--translate-y-01em su-transition-transform">
```

Comment:
In the icon, I replaced hardcoded 42px by 1em which seems to work out perfectly and it changes size proportionally if font size changes. Added the external link animation